src/ucs/sys/event_set.h

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (C) Hiroyuki Sato. 2019. ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCS_EVENT_SET_H
+#define UCS_EVENT_SET_H
+
+#include <ucs/type/status.h>
+
+/**
+ * ucs_sys_event_set structure used in ucs_event_set_XXX functions.
+ *
+ */
+typedef struct ucs_sys_event_set {
+    int                epfd;
+} ucs_sys_event_set_t;
```

Comment:
You said 
> this struct should be defined in C file, to avoid including config.h in this file in the future

This struct use in other function i.e. (ucs/async/thread.c)
Should I move this declaration in C file?
If so, Is it possible reference from that file(`ucs/async/thread.c`)?

Reply:
header file should have only typedef declaration, which is enough to use it by pointer. the struct itself should be in C file,